frontend/language/src/en.json

```diff
@@ -724,6 +724,7 @@
   "ux_editor.component_paragraph": "Paragraph",
   "ux_editor.component_radio_button": "Radio Button",
   "ux_editor.component_text_area": "Long answer",
+  "ux_editor.component_information_panel": "Panel",
```

Comment:
Maybe a little short and generic?
Could we come up with something more descriptive as for example with the Text Area being named "Long answer"
I think maybe "Static information" would be more descriptive. (I am not 100% sure how this is interacted with on the client side, but "Panel" can be pretty much anything.)

Reply:
Nice suggestion and I agree with you that "Panel" may be to generic name. What about "Static message" or "Message component"? 

The PR was approved and merged before I got the opportunity to notice your comment.